src/ontology/metadata/mondo.sssom.config.yml

```diff
@@ -4,6 +4,9 @@ curie_map:
   NCIT: http://purl.obolibrary.org/obo/NCIT_
   DOID: http://purl.obolibrary.org/obo/DOID_
   EFO: http://www.ebi.ac.uk/efo/EFO_
+  HGNC: "https://identifiers.org/hgnc:"
+  HGNC__2: "https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/HGNC:"
+  HGNC__symbol: https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/
```

Comment:
Use single underscore here as this may confuse the hacky code. Symbol and ID are different namespaces. I used the double underscore to mean same namespace different iri expansion..

Reply:
Makes perfect sense. I was wondering if this might be the case. Will make the change and push momentarily.